compiler-rt/lib/rtsan/tests/rtsan_test_interceptors_posix.cpp

```diff
@@ -421,14 +421,36 @@ TEST_F(RtsanFileTest, SetvbufDieWhenRealtime) {
   FILE *f = fopen(GetTemporaryFilePath(), "w");
   EXPECT_THAT(f, Ne(nullptr));
 
-  auto Func = [&f, &buffer, &size]() {
-    int r = setvbuf(f, buffer, _IOFBF, size);
+  auto Func = [f, buffer, size]() {
+    int r = setvbuf(f, (char *)buffer, _IOFBF, size);
```

Comment:
the cast shouldn't be needed

Reply:
otherwise it does not build for me

Reply:
I don't understand, seems to work for me: https://godbolt.org/z/xnGnWExn4